WalletWasabi.Fluent/ViewModels/Dialogs/Authorization/PasswordAuthDialogViewModel.cs

```diff
@@ -0,0 +1,38 @@
+using System.Threading.Tasks;
+using WalletWasabi.Blockchain.Transactions;
+using WalletWasabi.Fluent.ViewModels.Dialogs.Base;
+using WalletWasabi.Userfacing;
+using WalletWasabi.Wallets;
+
+namespace WalletWasabi.Fluent.ViewModels.Dialogs.Authorization
+{
+	[NavigationMetaData(Title = "Enter your password")]
+	public partial class PasswordAuthDialogViewModel : AuthorizationDialogBase
+	{
+		private readonly Wallet _wallet;
+		[AutoNotify] private string _password;
+
+		public PasswordAuthDialogViewModel(Wallet wallet)
+		{
+			_wallet = wallet;
+			_password = "";
+		}
+
+		protected override void OnDialogClosed()
+		{
+			Password = "";
+		}
+
+		protected override async Task Authorize()
```

Comment:
make this method return true or false, and the callee close the dialog... to avoid repeated code.

Reply:
done